examples/gno.land/r/demo/boards2/board.gno

```diff
@@ -131,14 +131,35 @@ func (board *Board) GetPostFormURL() string {
 	return txlink.Call("CreateThread", "bid", board.id.String())
 }
 
-// TODO: This is a temporary implementation until the permissions and DAO mecahnics are defined
 func createDefaultBoardPermissions(owner std.Address) *DefaultPermissions {
 	return NewDefaultPermissions(
-		admindao.New(admindao.WithMember(owner)),
+		admindao.New(),
```

Comment:
Removed member initialization because `WithUser(owner, RoleOwner)` now adds the user to the DAO if it's not a member.